core/domain/wipeout_service.py

```diff
@@ -1010,6 +1012,58 @@ def _remove_user_id_from_models_transactional(summary_models):
                 i:i + feconf.MAX_NUMBER_OF_OPS_IN_TRANSACTION])
 
 
+def _pseudonymize_app_feedback_report_models(pending_deletion_request):
+    """Pseudonymize the app feedback report models for the user with user_id
```

Comment:
Could you add some notes in this docstring to explain whether the intention is, if the same user scrubs several reports, to give them the same pseudonym across those reports, or to treat each report independently? Please also ensure that there are backend tests that validate whatever choice you make.

Also /cc @vojtechjelinek for a review of this part.

Reply:
Done, using the same pseudonym for the same user in all entities

To do this involved manually checking and creating the model entity category's mapping to pseudonyms in the deletion request since the existing `_save_pseudonymizable_entity_mappings` function ([here](https://github.com/oppia/oppia/pull/12298/files#diff-45edc7eecb4767d3bbacbcf661c5f690c45fe3668674845ff8458ccbe0d967cbL507)) maps each entity to a different pseudonym.